apps/intellij/build.gradle.kts

```diff
@@ -173,12 +173,16 @@ tasks {
 }
 
 tasks.register<Exec>("buildNxls") {
-    commandLine = listOf("bash", "-c", "npx nx run nxls:build")
+    commandLine = buildCommands().apply {
+        add("npx nx run nxls:build")
+    }
     workingDir = rootDir
 }
 
 tasks.register<Exec>("buildGenerateUi") {
-    commandLine = listOf("bash", "-c", "npx nx run generate-ui:build:production-intellij")
+    commandLine = buildCommands().apply {
```

Comment:
Maybe this can be the same as
`buildCommands().plus("npx nx run generate-ui:build:production-intellij")`